Presentation.Web/Tests/Organization/UserOverview.e2e.spec.ts

```diff
@@ -19,19 +19,26 @@ describe("Only Global Admins can enable and disable API access on a user", () =>
         testFixture.cleanupState();
     });
 
-    it("Global Admin only have access to API access attribute in overview", () =>
+    it("Global Admin can see API access attribute in overview", () =>
     {
         loginHelper.loginAsGlobalAdmin();
         pageObject.getPage();
         browser.wait(ec.presenceOf(pageObject.kendoToolbarWrapper.columnHeaders().userApi), waitUpTo.twentySeconds);
         expect(pageObject.kendoToolbarWrapper.columnHeaders().userApi.isDisplayed()).toBeTruthy();
     });
 
-    it("Local Admin and down cannot see API access attribute in overview", () =>
-    {
+    it("Local Admin can see API access attribute in overview", () => {
         loginHelper.loginAsLocalAdmin();
         pageObject.getPage();
         browser.wait(ec.presenceOf(pageObject.kendoToolbarWrapper.columnHeaders().userApi), waitUpTo.twentySeconds);
+        expect(pageObject.kendoToolbarWrapper.columnHeaders().userApi.isDisplayed()).toBeTruthy();
+    });
+
+    it("Regular user cannot see API access attribute in overview", () =>
+    {
+        loginHelper.loginAsRegularUser();
+        pageObject.getPage();
+        browser.wait(ec.presenceOf(pageObject.kendoToolbarWrapper.columnHeaders().userApi), waitUpTo.twentySeconds);
         expect(pageObject.kendoToolbarWrapper.columnHeaders().userApi.isDisplayed()).toBeFalsy();
```

Comment:
Denne er som de to andre på nær den sidste condition. Gør derfor metoden i stand til at understøtte begge scenarier ved at tage en parameter med